widget/Toaster.js

```diff
@@ -158,28 +162,32 @@ define([
 				this.slideAnim = coreFx.slideTo({
 					node: this.containerNode,
 					top: 0, left: 0,
-					duration: this.slideDuration});
-				this.connect(this.slideAnim, "onEnd", function(){
+					duration: this.slideDuration,
+					onEnd: lang.hitch(this, function(){
 						//we build the fadeAnim here so we dont have to duplicate it later
 						// can't do a fadeHide because we're fading the
 						// inner node rather than the clipping node
-						this.fadeAnim = baseFx.fadeOut({
+						this.fadeAnim = (baseFx.fadeOut({
 							node: this.containerNode,
-							duration: 1000});
-						this.connect(this.fadeAnim, "onEnd", function(){
-							this.isVisible = false;
-							this.hide();
-						});
+							duration: 1000,
+							onEnd: lang.hitch(this, function(){
+								this.isVisible = false;
+								this.hide();
+							})
+						}));
+						this.own(this.fadeAnim);
 						this._setHideTimer(duration);
-						this.connect(this, 'onSelect', function(){
+						this.own(aspect.after(this, 'onSelect', lang.hitch(this, function(){
```

Comment:
Tried that earlier and was getting  Error: Target must be an event emitter notifications, will try again.